supernovacontroller/sequential/i2c.py

```diff
@@ -252,6 +252,50 @@ def write(self, address, register, data):
 
         return result
 
+    def write_non_stop(self, address, register, data):
+        """
+        Performs a write operation to a specified register on an I2C device, without issuing a stop condition at the end.
+
+        This method sends data to a specified register of a device with a given I2C address, without issuing a stop condition at the end.
+        Before performing the write operation, it checks if the bus voltage is initialized.
```

Comment:
> Before performing the write operation, it checks if the bus voltage is initialized.

Does this happen at the SDK level, should it be added to the code, or is the docstring outdated and should be removed?